tests/ui/contracts/cross-crate-checks/auxiliary/id.rs

```diff
@@ -0,0 +1,16 @@
+//@ [unchk_pass] compile-flags: -Zcontract-checks=no
+//@ [unchk_fail] compile-flags: -Zcontract-checks=yes
+//@ [chk_pass] compile-flags: -Zcontract-checks=no
+//@ [chk_fail] compile-flags: -Zcontract-checks=yes
```

Comment:
I didn't realize it was possible to make aux tests also handle revisions from the aux-test user. Considering the main test properly crashes or passes in all the cases configured here, it does seem to work flawlessly